lib/extended_button.dart

```diff
@@ -27,10 +27,14 @@ import 'package:flutter/material.dart';
 ///
 /// [boxColor] is used to define the color for the Extended Button. By default value of
 /// [boxColor] is [Colors.black]
+///
+/// [isRounded] is used to define the Border Radius for the Extended Button. By default,
```

Comment:
Few nits, change the documentation to this.

[isRounded] is used to define the `borderRadius` for the ExtendedButton when button is in the shrunk state. By default, the value of [isRounded] is false. When `isRounded` is true, the value of `borderRadius` is 5.0.